### PR TITLE
[ru] add translate for DataView.prototype.byteLength

### PR DESCRIPTION
**Pull request by 42proger**
### Description

Translated: DataView.prototype.byteLength article translated into Russian language.

### Motivation

This article was not originally translated to Russian language.

### CODE REVIEW

File: files/ru/web/javascript/reference/global_objects/dataview/bytelength/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+{{JSRef}}
+
+Свойство **`byteLength`** возвращает длину объектов {{jsxref("DataView")}} в байтах.
+
+{{EmbedInteractiveExample("pages/js/dataview-bytelength.html")}}
+
+## Описание
+
+Свойство `byteLength` - это свойством доступа, функция set accessor которого `undefined`, что означает, что вы можете только читать это свойство. Значение устанавливается при создании объекта `DataView` и не может быть изменено. Если `DataView` не указывает смещение или длину (`byteLength`), то будет возвращена `byteLength` указанного `ArrayBuffer` или `SharedArrayBuffer`.
```
Comment (Saionaro) [inline files/ru/web/javascript/reference/global_objects/dataview/bytelength/index.md:19]:
```suggestion
Свойство `byteLength` доступно только для чтения. Его значение определяется при создании объекта `DataView` и не может быть изменено. Если при создании `DataView` не были переданы параметры `byteOffset` или `byteLength`, то свойство будет равно `byteLength` связанного `ArrayBuffer` или `SharedArrayBuffer`.
```

File: files/ru/web/javascript/reference/global_objects/dataview/bytelength/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Примеры
+
+### Использование свойства byteLength
+
+```js
+const buffer = new ArrayBuffer(8);
+const dataview = new DataView(buffer);
+dataview.byteLength; // 8 (соответствует длине буфера в байтах)
```
Comment (Saionaro) [inline files/ru/web/javascript/reference/global_objects/dataview/bytelength/index.md:28]:
```suggestion
dataview.byteLength; // 8 (соответствует byteLength буфера)
```

File: files/ru/web/javascript/reference/global_objects/dataview/bytelength/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+const dataview2 = new DataView(buffer, 1, 5);
+dataview2.byteLength; // 5 (как указано при построении DataView)
```
Comment (Saionaro) [inline files/ru/web/javascript/reference/global_objects/dataview/bytelength/index.md:31]:
```suggestion
dataview2.byteLength; // 5 (как указано при создании DataView)
```

File: files/ru/web/javascript/reference/global_objects/dataview/bytelength/index.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+const dataview3 = new DataView(buffer, 2);
+dataview3.byteLength; // 6 (из-за смещения при построении DataView)
```
Comment (Saionaro) [inline files/ru/web/javascript/reference/global_objects/dataview/bytelength/index.md:34]:
```suggestion
dataview3.byteLength; // 6 (из-за byteOffset, заданного при создании DataView)
```

File: files/ru/web/javascript/reference/global_objects/dataview/bytelength/index.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+tags:
+  - DataView
+  - JavaScript
+  - byteLength
+translation_of: Web/JavaScript/Reference/Global_Objects/DataView/byteLength
+---
+
+{{JSRef}}
+
+Свойство **`byteLength`** возвращает длину объектов {{jsxref("DataView")}} в байтах.
```
Comment (Saionaro) [inline files/ru/web/javascript/reference/global_objects/dataview/bytelength/index.md:13]:
```suggestion
Свойство-геттер **`byteLength`** возвращает длину {{jsxref("DataView")}} в байтах.
```

**Review by saionaro — APPROVED** [review]
@42proger привет! Принимаю PR с небольшими правками 👍 
Спасибо за вклад в MDN!

**Pull request merged**